Configuration/TypoScript/setup.txt

```diff
@@ -55,6 +55,23 @@ plugin.tx_dpf {
 			searchResultsPid = {$plugin.tx_dpf.settings.searchResultsPid}
         }
 }
+// pi1 = GndAjax
+// PAGE object for Ajax call:
+search_page = PAGE
+search_page {
+	typeNum = 427590
+
+	config {
+		disableAllHeaderCode = 1
+		additionalHeaders = Content-type:application/json
+		xhtml_cleaning = 0
```

Comment:
xhtml_cleaning does not exist in TYPO3 7.6 anymore.

'0' is no documented setting. Do not set this config, if you don't want to use the feature.